package.json

```diff
@@ -61,7 +61,7 @@
     "commander": "^2.9.0",
     "cookie-parser": "^1.4.3",
     "cookie-session": "^2.0.0-beta.3",
-    "db-migrate": "0.10.0-beta.24",
+    "db-migrate": "0.10.0",
```

Comment:
can we add `^` as well, then?

Reply:
Yes that should be ok! So far I have had good experience with the versioning of this module.